salt/resource-manager/init.sls

```diff
@@ -40,7 +44,7 @@ resource-manager_log:
     - makedirs: True
     - dir_mode: 755
 
-resource-manager_spark_common_wrapper:
+resource-manager_spark_flink_common_wrapper:
```

Comment:
why did you rename the state id with Flink, worth keeping the original one. thx

Reply:
i have renamed the state with flink, as it will be the wrapper for both  spark and flink. if you want so will rename it as original ?

Reply:
ok let's keep Flink too then